fixture-monkey-api/src/main/java/com/navercorp/fixturemonkey/api/plugin/SimpleValueJqwikPlugin.java

```diff
@@ -141,10 +141,10 @@ public void accept(FixtureMonkeyOptionsBuilder optionsBuilder) {
 							new SimpleJavaConstraintGenerator(
 								this.minStringLength,
 								this.maxStringLength,
-								this.positiveMinNumberValue,
-								this.positiveMaxNumberValue,
 								this.negativeMinNumberValue,
 								this.negativeMaxNumberValue,
+								this.positiveMinNumberValue,
+								this.positiveMaxNumberValue,
```

Comment:
For keeping consistency with `JavaDecimalConstraint`, `JavaIntegerConstraint` , it is better to be positive first and then negative.

Could you fix the order of the fields in `SimpleJavaConstraintGenerator` ?

Reply:
Sure. I'm going to try that, and give you notie :)